src/commands/farm.rs

```diff
@@ -101,26 +105,41 @@ pub(crate) async fn farm(is_verbose: bool, executor: bool) -> Result<()> {
 
 #[instrument]
 async fn wait_on_farmer(
-    maybe_handles: Option<(JoinHandle<()>, JoinHandle<()>)>,
+    mut maybe_handles: MaybeHandles,
     farmer: Arc<Farmer>,
```

Comment:
I think you no longer need an `Arc` for a farmer

Reply:
I still do unfortunately. 
Plotting subscription needs `farmer`.
`farmer` being not `Clone`, it tries to move `Farmer`, and I also need to close `Farmer` after this subscription